.gitignore

```diff
@@ -1,3 +1,7 @@
 build/
 node_modules/
 *.sublime*
+.idea
+
+*.d.ts
+*.js
```

Comment:
Did you really check tslint ? I guess new line is expected in tslint

Reply:
tslint do not check gitignore